src/presentation/api/attendances.ts

```diff
@@ -56,7 +56,7 @@ export class AttendanceAPI extends Controller {
   @SuccessResponse(200, 'Successfully got attendances')
   public async getAttendancesFromDate(
     @Path() date: string
-  ): Promise<AttendanceEntity[]> {
+  ): Promise<(AttendanceEntity & { name: string | null })[]> {
```

Comment:
I was thinking about adding the name field directly on the attendance entity, which I think makes sense with the business logic. And the only cases that we would need to remove the name field would be when creating or updating the attendance, as we will only modify the idvol field.

Do you think that it would be a good aproach?

Reply:
I chose this approach to keep the AttendanceEntity entity according to the model.
This way, we can use it as a basis for several joins.
I thought about creating an entity with the name attribute for this case.
What do you think?

Reply:
It makes sense, I think an entity with the name would be great, as you suggested

Reply:
done! thanks